pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -8,16 +8,15 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"net/url"
 	"regexp"
 	"sort"
 	"strings"
 	"sync"
 	"time"
 
 	corev1 "k8s.io/api/core/v1"
-	apixv1beta1client "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
 	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	apixv1beta1client "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
```

Comment:
use `v1`

Reply:
This change is completely unrelated to the PR; just a reshuffle of the import order, but I'll fix it anyways because it wasn't backported (yet).